_pages/business-units/18f/leadership-resources/leadership-selection.md

```diff
@@ -19,33 +19,33 @@ This guide is meant to help us create an inclusive and equitable process for sel
 
    * Who is being nominated?
    * Why would they be good in the role?
-   * Anything else to add (optional)
+   * Anything else to add (optional)?
 
    Be sure that the form is set to capture the email address of the nominator in case you need to follow up.
 
-3. Set your deadline. Your deadline should be at least five business days out, preferably longer. For roles with formal applications (e.g., supervisors, chapter leads, etc.), your deadline should include a weekend. For roles that require submitting a government style resume, extend the deadline to allow people time to update their resumes and get feedback from friends. Scheduling deadlines for end-of-day Pacific Time Friday is pretty convenient and easy to remember for everyone.
+3. Set your deadline. Your deadline should be at least five business days out, preferably longer. For roles with formal applications (e.g., supervisors, chapter leads, etc.), your deadline should include a weekend. For roles that require submitting a government-style résumé, extend the deadline to allow people time to update their résumés and get feedback from friends. Scheduling deadlines for end-of-day Friday (Pacific Time) is convenient and easy to remember.
```

Comment:
I don't know if we have specific guidance on résumé or not, but added for now. Added one compound adjective and separated another.